rank/data/tests/works.json

```diff
@@ -0,0 +1,439 @@
+                    "xwtcsk93"
+                ],
+                "description": "Example of a known title's prefix, but not the full thing",
+                "knownFailure": true
+            },
+            {
+                "searchTerms": "L0033046",
+                "ratings": [
+                    "kmebmktz"
+                ],
+                "description": "Miro ID matching"
+            },
+            {
+                "searchTerms": "kmebmktz",
+                "ratings": [
+                    "kmebmktz"
+                ],
+                "description": "Work ID matching"
+            },
+            {
+                "searchTerms": "gzv2hhgy",
+                "ratings": [
+                    "kmebmktz"
+                ],
+                "description": "Image ID matching"
+            },
+            {
+                "searchTerms": "Oxford dictionary of national biography",
+                "ratings": [
+                    "ruedafcw"
+                ],
+                "description": "Example of a known title's prefix, but not the full thing",
+                "knownFailure": true
+            }
+        ],
+        "metric": {
+            "precision": {
+                "relevant_rating_threshold": 3,
+                "k": 1
+            }
+        }
+    },
+    {
+        "id": "recall",
+        "description": "Ensure that the query brings back the results we expect somewhere in the list",
+        "eval": "equalTo1",
+        "cases": [
+            {
+                "searchTerms": "Atherosclerosis: an introduction to atherosclerosis",
+                "ratings": [
+                    "bcwvtknn",
+                    "rty8296y"
+                ],
+                "description": "Two works with matching titles"
+            },
+            {
+                "searchTerms": "2013i 2599i",
+                "ratings": [
+                    "djmjw2cu",
+                    "xxskepr5"
+                ],
+                "description": "Multiple IDs"
+            },
+            {
+                "searchTerms": "wa/hmm durham",
+                "ratings": [
+                    "euf49qkx",
+                    "tpxy78kr",
+                    "gu3z98y4",
+                    "ad3rfubw"
+                ],
+                "description": "Archive refno and a word from the title"
+            },
+            {
+                "searchTerms": "wa/hmm benin",
+                "ratings": [
+                    "qfdvkegw",
+                    "je5pm2gj",
+                    "dppjjtqz"
+                ],
+                "description": "Archive refno and a word from the description"
+            },
+            {
+                "searchTerms": "WA/HMM/CM benin",
+                "ratings": [
+                    "qfdvkegw",
+                    "je5pm2gj",
+                    "dppjjtqz"
+                ],
+                "description": "Archive refno and a word from the description"
+            },
```

Comment:
Extreme nitpick: what you're testing is the Calm <strong>Alt</strong>RefNo, not the RefNo.

The RefNo is the absolutely structured thing we use to create the archive trees, which in this case would be `WAHMM/CM`, and should never change. The AltRefNo is the publicly visible thing, `WA/HMM/CM`, which can sometimes change. It's for display purposes only.

We have two because the first prefix before a slash serves two different purposes:

* In the RefNo it identifies everything inside this archive; everything matching `WAHMM/*` is part of the Wellcome Historical Medical Museum. Everything starting `WA/*` would be too broad to be useful.
* In the AltRefNo it tells the reader what sort of archive this is, e.g. `PP/*` is personal papers (I don't remember what `WA/*` is)

I would suggest:

1. Change this description to say `archive reference`
2. Add a test for the RefNo, which is `WAHMM/CM`. Probably only used by staff, but should still work in results.